### DISCUSSION TITLE
How to I run scinterm?

### DISCUSSION THREAD

**Discussion by Zciurus-Alt-Del** [orbitalquark/scinterm#9] [Q&A] [answered]
I sucessfully compiled scinterm and now have the file `bin/scintilla.a`. How can I run scinterm now?
I don't have a lot of expierence with *nix, so pardon if I'm missing something obvious.

**Comment by orbitalquark** [comment, marked as answer]
You can compile the *jinx/* sample. Scinterm is just a curses/terminal interface to Scintilla. You still need to write an application to harness it.
